kooplearn/nn/functional.py

```diff
@@ -220,36 +236,38 @@ def eym_score_split(
         loss_on = (
             0.5
             * (
-                torch.trace(uc_cov_U1 @ uc_cov_U2)
+                torch.sum(uc_cov_U1 @ uc_cov_U2)
                 - torch.trace(uc_cov_U1)
                 - torch.trace(uc_cov_U2)
-                + torch.trace(uc_cov_V1 @ uc_cov_V2)
+                + torch.sum(uc_cov_V1 @ uc_cov_V2)
                 - torch.trace(uc_cov_V1)
                 - torch.trace(uc_cov_V2)
             )
             + d
         )
-        return score - metric_deformation * loss_on
+        return -1 * (score + metric_deformation * loss_on)
     else:
-        return score
-
+        return -1 * score
+    
 
 def eym_score(
```

Comment:
Are there any chances that the `Ustat` mode will ever work? If not, just remove it, and every reference to it in the code.